src/providers/pronouns/UserPronouns.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <QString>
+
+namespace chatterino::pronouns {
+
+class UserPronouns
+{
+public:
+    UserPronouns() = default;
+    UserPronouns(QString);
+
+    QString format() const
+    {
+        if (!this->representation.isNull())
```

Comment:
This should use either `isUnspecified` or `operator bool` to reduce code duplication (same goes for one of the two, feel free to choose).